employees/tests/test_report_detail_and_delete_views.py

```diff
@@ -103,16 +103,16 @@ def test_project_report_detail_view_should_show_data_from_current_report_not_fro
         ReportFactory(author=self.user)
         response = self.client.get(path=reverse("custom-report-detail", args=(self.report.pk,)))
         self.assertEqual(response.status_code, 200)
-        self.assertEqual(response.context_data["form"].initial["task_activities"], self.report.task_activities.pk)
-        self.assertEqual(response.context_data["form"].initial["project"], self.report.project.pk)
+        self.assertEqual(response.context_data["form"].initial["task_activities"], self.report.task_activities)
```

Comment:
I see that `Report` model has wrong name for `TaskActivityType`. Please in separate commit change name for `task_activity` instead of `task_activities`.

Reply:
Changed name to task_activity. Appropriate changes applied to other files.